src/EnergyPlus/HeatBalanceManager.cc

```diff
@@ -108,6 +108,7 @@
 #include <EnergyPlus/WindowComplexManager.hh>
 #include <EnergyPlus/WindowEquivalentLayer.hh>
 #include <EnergyPlus/WindowManager.hh>
+#include <EnergyPlus/ZoneEquipmentManager.hh>
```

Comment:
And this include is no longer needed.

Reply:
Oops...I knew I was going to forget something small like that.  Taken care of in the new commit just made.  Thanks for spotting that.